internal/storagetools/check.go

```diff
@@ -0,0 +1,61 @@
+package storagetools
+
+import (
+	"bytes"
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	"github.com/wal-g/wal-g/pkg/storages/storage"
+
+	"github.com/wal-g/tracelog"
+)
+
+func HandleCheckRead(folder storage.Folder, filenames []string) {
```

Comment:
Maybe name it like `check exists`? Because it seems like the actual behavior of this command.

Reply:
file existance check is optional